core/tests/account_service_tests.rs

```diff
@@ -138,4 +149,23 @@ fn import_account_public_key_mismatch() {
     ));
 }
 
-// TODO add a test for asserting metadata version of root is non zero
+#[test]
+fn export_account() {
+    let core = test_core();
+    core.create_account(&random_name(), &url()).unwrap();
+    core.export_account().unwrap();
+}
+
+#[test]
+fn export_account_qr() {
+    let core = test_core();
+    core.create_account(&random_name(), &url()).unwrap();
+    core.export_account_qr().unwrap();
+}
```

Comment:
Can exporting qr just be added as the last line to the `export_account` test above?

Reply:
done!